### PR TITLE
Add 'launch' to sets of launch file extensions

### PR DESCRIPTION
**Pull request by christophebedard**
Closes https://github.com/ros2/launch_ros/issues/149

Alternative to https://github.com/ros2/launch_ros/pull/251 after https://github.com/ros2/launch/pull/516, but won't do anything until https://github.com/ros2/launch_ros/pull/252 is merged.

Signed-off-by: Christophe Bedard <bedard.christophe@gmail.com>

### CODE REVIEW

File: launch_yaml/launch_yaml/parser.py [hunk 1]
```diff
@@ -57,4 +57,4 @@ def load(
     @classmethod
     def get_file_extensions(cls) -> Set[Text]:
         """Return the set of file extensions known to this parser."""
-        return {'launch.yaml', 'launch.yml', 'yaml', 'yml'}
+        return {'launch.yaml', 'launch.yml', 'yaml', 'yml', 'launch'}
```
Comment (hidmic) [inline launch_yaml/launch_yaml/parser.py:60]: @christophebedard meta: one that we probably missed in #516 is that e.g. `yaml` already includes `launch.yaml` making the latter redundant.

**Review by hidmic — APPROVED** [review]
LGTM pending green CI (I'll stay put till https://github.com/ros2/launch/pull/518 is in)

File: launch_yaml/launch_yaml/parser.py (hunk 1 above)
Comment (christophebedard) [reply launch_yaml/launch_yaml/parser.py:60]: yeah I noticed the redundancy too :laughing:

I simply thought it didn't really change anything in practice and it acts as an explicit list of supported file extensions even if it's redundant, but I can remove the redundant ones if you want. Want me to remove them?

Comment (hidmic) [reply launch_yaml/launch_yaml/parser.py:60]: No biggie. We can remove them in a follow-up PR.

**Comment by hidmic** [comment]
@christophebedard this one looks ready for a CI run, doesn't it?

**Comment by christophebedard** [comment]
`--packages-above launch launch_xml launch_yaml`:

* Linux [Build Status](http://ci.ros2.org/job/ci_linux/15100/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/9832/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/12787/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/15268/)

**Comment by hidmic** [comment]
Alright, CI's happy. Going in! Thanks @christophebedard.

**Pull request merged**